hpx/parallel/execution_policy.hpp

```diff
@@ -1418,6 +1423,11 @@ namespace hpx { namespace parallel { HPX_INLINE_NAMESPACE(v1)
         template <>
         struct is_parallel_execution_policy<parallel_task_execution_policy>
           : std::true_type
+        {};        
+
+        template <typename Executor, typename Parameters>
+        struct is_parallel_execution_policy<parallel_task_execution_policy_shim<Executor, Parameters>>
+          : std::true_type
         {};
         /// \endcond
```

Comment:
Now 'parallel' are parallel, parallel_shim, parallel_task, parallel_task_shim and parallel_vector.
Which other am I missing in is_parallel_execution_policy? Or did you mean something else by 'add'?